### DISCUSSION TITLE
Suggested documentation change

### DISCUSSION THREAD

**Discussion by burlesona** [jeremyevans/sequel#1824] [General]
Hi Jeremy,

I ran into an issue today where I was testing validations and was surprised that calling `save` raised an error on invalid records rather than returning `false` as in AR. I was confused because I did not think I had set `raise_on_save_failure` to true, and could not find that anywhere in my codebase. Then I dug around the docs a while and finally realized that this defaults to true. So, nothing wrong, I just remembered the default incorrectly.

I did want to offer a small suggestion for you to consider...

The first docs I checked were the [save method](https://www.rubydoc.info/github/jeremyevans/sequel/Sequel%2FModel%2FInstanceMethods:save).

Those docs are clear that `raise_on_save_failure` determines the default, but they don't mention what the default value for that setting is, nor do they link to [the docs for it](https://www.rubydoc.info/github/jeremyevans/sequel/Sequel/Model/ClassMethods#raise_on_save_failure-instance_method).

Would it make sense to you to either add mention of the default value on the `save` docs, or add a link to `raise_on_save_failure`?

**Comment by jeremyevans** [comment]
The documentation for `save` does not link to other methods that affect `save`, such as `require_modification` or `use_transactions`.  Seems inconsistent to link to `raise_on_save_failure` considering that.  However, I don't necessarily object to doing so.  Please send a pull request if you would like that to happen.
